deploy/ray-operator/README.md

```diff
@@ -0,0 +1,217 @@
+# Ray-Operator Documentation
+
+To introduce the Ray-Operator, give an explanation of RayCluster CR firstly.  
+If interested in CRD, refer to file deploy/ray-operator/config/crd/bases/ray.io_rayclusters.yaml for more detail.  
+If interested in RayCluster types, refer to file deploy/ray-operator/api/v1/raycluster_types.go.  
+
+## RayCluster sample CR
+4 pods in this sample, 1 for head and 3 for workers but with different specifications.
```

Comment:
Hi @Qstar , can you introduce more about the meanings and usage of files/directories under deploy/ray-operator ?
And CR example can be reference by link, no need to paste the details in this document.
Also, I think you need to add a step by step guide for user to submit a RayCluster.

Reply:
- added directories structure explanation
- added  reference by link by relative file
- added software requirement and guide for submitting